tests/unit/routes/addons/ember-tooltips-test.js

```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:addons/ember-tooltips', 'Unit | Route | addons/ember tooltips', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+test('it exists', function(assert) {
```

Comment:
Shouldn't be adding tests without filling them out. Can we delete theses before the merge?